helm/dscp-identity-service/Chart.yaml

```diff
@@ -1,20 +1,20 @@
 apiVersion: v2
 name: dscp-identity-service
-appVersion: '1.7.1'
+appVersion: '1.7.2'
 description: A Helm chart for dscp-identity-service
-version: '1.7.1'
+version: '1.7.2'
 type: application
 maintainers:
   - name: digicatapult
     email: opensource@digicatapult.org.uk
     url: www.digicatapult.org.uk
 dependencies:
   - name: postgresql
-    version: '^10.12.2'
+    version: '^11.9.10'
     repository: https://charts.bitnami.com/bitnami
     condition: postgresql.enabled
   - name: dscp-node
-    version: '4.2.1'
+    version: '4.2.2'
```

Comment:
there's now a chart for `4.3.1` if we want the helm dep to match what's in `package.json`

Reply:
done